Tasks/Gradle/sonarqube-common/restresponse.ts

```diff
@@ -0,0 +1,13 @@
+// Data class representing an HTTPS response from the SonarQube server.
+export class RestResponse {
+    constructor(public statusCode:number, public payload:string) {
+    }
+
+    public wasSuccess():boolean {
+        return this.statusCode == 200;
```

Comment:
200 is not the only success status code, all of the codes between 200 and 299 are success. See http://stackoverflow.com/questions/32569860/checking-if-httpstatuscode-represents-success-or-failure or wikipedia. Not sure if you really this level of detail